desktop/src/views/Pro/Workspace/Logs.tsx

```diff
@@ -144,18 +149,161 @@ type TActionTerminalProps = Readonly<{
 }>
 function ActionTerminal({ actionID }: TActionTerminalProps) {
   const action = useAction(actionID)
-  const { terminal, connectStream, clear: clearTerminal } = useStreamingTerminal()
+
+  const [searchString, setSearchString] = useState<string | undefined>(undefined)
```

Comment:
Could you wrap these controls into a separate hook/component and also add them to the open source logs please? 
`views/Actions/Action.tsx`

Reply:
Done